docs/api-reference/loaders-storage-targets/lake-loader/index.md

```diff
@@ -17,7 +17,7 @@ The Lake Loader is an application that loads Snowplow events to a cloud storage
 
 :::info Open Table Formats
 
-The Lake Loader supports the three major Open Table Formats: [Delta](https://delta.io/), [Iceberg](https://iceberg.apache.org/) and [Hudi](https://hudi.apache.org/).
+The Lake Loader supports the two major Open Table Formats: [Delta](https://delta.io/) and [Iceberg](https://iceberg.apache.org/).
```

Comment:
Or

- The two most major
- The two majorest (made up word)
- The two Open Table Formats that Ian likes
- Both sensible Open Table Formats